README.md

```diff
@@ -322,20 +322,28 @@ The schemas mirror the CDF API as closely as possible.
 | `dataSetId`       | `long`                | Yes       |
 
 ### Files schema
-| Column name       | Type                  |  Nullable |
-| ------------------| ----------------------| --------- |
-| `id`              | `long`                | No        |
-| `name`            | `string`              | No        |
-| `source`          | `long`                | Yes       |
-| `externalId`      | `string`              | Yes       |
-| `mimeType`        | `string`              | Yes       |
-| `metadata`        | `map(string, string)` | Yes       |
-| `assetIds`        | `array(long)`         | Yes       |
-| `uploaded`        | `boolean`             | No        |
-| `uploadedTime`    | `timestamp`           | Yes       |
-| `createdTime`     | `timestamp`           | No        |
-| `lastUpdatedTime` | `timestamp`           | No        |
-| `uploadUrl`       | `string`              | Yes       |
+| Column name          | Type                  |  Nullable |
+| ---------------------| ----------------------| --------- |
+| `id`                 | `long`                | No        |
+| `name`               | `string`              | No        |
+| `source`             | `long`                | Yes       |
+| `externalId`         | `string`              | Yes       |
+| `mimeType`           | `string`              | Yes       |
+| `metadata`           | `map(string, string)` | Yes       |
+| `assetIds`           | `array(long)`         | Yes       |
+| `uploaded`           | `boolean`             | No        |
+| `uploadedTime`       | `timestamp`           | Yes       |
+| `createdTime`        | `timestamp`           | No        |
+| `lastUpdatedTime`    | `timestamp`           | No        |
+| `sourceCreatedTime`  | `timestamp`           | Yes       |
+| `sourceModifiedTime` | `timestamp`           | Yes       |
+| `securityCategories` | `array(long)`         | Yes       |
+| `uploadUrl`          | `string`              | Yes       |
+| `dataSetId`          | `long`                | Yes       |
+
+
+
+
```

Comment:
Remove those newlines ;)

Reply:
Not that important, you can just skip this if the tests pass now. The events thing was unrelated, maybe we need to adjust that test, but it's clearly not because of your changes.